index.html

```diff
@@ -318,7 +323,13 @@ <h2>
       </h2>
       <p>
         This interface defines the state of an individual button on a gamepad
-        device.
+        device. A GamepadButton is a live object; that is, user interaction with
```

Comment:
```suggestion
        device. A {{GamepadButton}} is a live object; that is, user interaction with
```

Reply:
I think this should just read:
"A {{GamepadButton}} is a live object: When the system receives new data from a connected gamepad..." 

And drop the "that is, user interaction with....". That makes it more clear that updates to the various properties are dependent on the queued microtasks.